### PR TITLE
Multi-datasource dashboards & instructions to load dashboards in Grafana config

### PR DESCRIPTION
**Pull request by jkaflik**

### CODE REVIEW

File: src/dashboards/cluster-analysis.json [hunk 1]
```diff
@@ -1,14 +1,5 @@
 {
```
Comment (bossinc) [inline src/dashboards/cluster-analysis.json:1]: What do these changes do?

Comment (jkaflik) [reply src/dashboards/cluster-analysis.json:1]: Not sure why you are asking. There is no formatting changes, only actual changes.

File: README.md [hunk 2]
```diff
@@ -154,9 +154,22 @@ on the Dashboards tab.
 
 We distribute the following dashboards with the plugin. These are aimed at assisting with support analysis of a ClickHouse cluster and do not rely on external datasets. The querying user requires access to the `system` database.
 
-1. Cluster Analysis - an overview of configured clusters, merges, mutations and data replication.
-2. Data Analysis - an overview of current databases and tables, including their respective sizes, partitions and parts.
-3. Query Analysis - an analysis of queries by type, performance and resource consumption.
+1. [Cluster Analysis](src/dashboards/cluster-analysis.json) - an overview of configured clusters, merges, mutations and data replication.
+2. [Data Analysis](src/dashboards/data-analysis.json) - an overview of current databases and tables, including their respective sizes, partitions and parts.
+3. [Query Analysis](src/dashboards/query-analysis.json) - an analysis of queries by type, performance and resource consumption.
```
[the remark this answers is not in the document]
Comment (jkaflik) [reply README.md:159]: Perhaps README should explain this way of importing?

[the remark this answers is not in the document]
Comment (kgoralski) [reply README.md:159]: Yes, I'm currently doing it with the links to github dashboard jsons. 
Could be simpler to make those dashboards available on grafana.com and import them by ID?

**Comment by bossinc** [comment]
@jkaflik I do not think adding links to the premade dashboards in the docs is helpful. The docs already described how to add the premade dashboards. Please remove the README.md edits and I'll approve this PR. :slightly_smiling_face:

**Comment by kgoralski** [comment]
> @jkaflik I do not think adding links to the premade dashboards in the docs is helpful. The docs already described how to add the premade dashboards. Please remove the README.md edits and I'll approve this PR. 🙂

imo that's helpful, it's a quick info how to include the dashboards for this plugin. Without this readme edits is not clear how to do that and you need few additional link clicks to learn about it. And the import dashboards docs are general docs and not specific ones from here. 

With readme edits it's self contained and less confusing for the reader.

**Comment by jkaflik** [comment]
README revert

**Comment by jkaflik** [comment]
@bossinc @asimpson can you folks have a look? thanks

**Review by asimpson — APPROVED** [review]
Nice work on this! :rocket:

**Comment by asimpson** [comment]
@jkaflik I'm happy to merge this but I wanted to check with you, anything else you wanted to add?

**Comment by jkaflik** [comment]
@asimpson all good. let's merge

**Pull request merged**